### DISCUSSION TITLE
가능한 날에는 팀미팅을 오전으로!

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#130] [Polls]
현직 대학생 @gayeong0916 님의 시간표 상, 우리 팀의 데일리 스크럼이 3시가 되었는데요.
프로젝트 막바지라 가능한 요일만이라도 오전에 스크럼을 하고 모각코를 일찍 시작하면 어떨까 합니다.
가영님 시간표로는 수, 목은 오전에 참여 가능하다고 하시네요! 의견 부탁 드립니다

**Comment by JuhyeokC** [comment]
좀 더 이른 시간이 추가됐군요!

혹시 더 이른 시간에 참여가 어려우신 분들은 의견 남겨주시면 좋겠네요!
